artpaint/controls/ColorFloatSlider.cpp

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright 2022, Dale Cieslak
+ * Distributed under the terms of the MIT License.
+ *
+ * Authors:
+ *		Dale Cieslak <dcieslak@yahoo.com>
+ *
+ */
+
+#include "ColorFloatSlider.h"
+
+#include "FloatControl.h"
+
+
+#include <GroupLayoutBuilder.h>
+#include <Message.h>
+#include <String.h>
+#include <Window.h>
+
+
+#include <new>
+#include <stdlib.h>
+
+
+namespace ArtPaint {
+	namespace Interface {
+
+enum {
+	kNumberControlFinished		= 'kncf',
+	kSliderValueModified		= 'ksvm',
+	kSliderModificationFinished	= 'ksmf'
+};
+
+
+ColorFloatSlider::ColorFloatSlider(const char* label, const char* text,
+		BMessage* message, float minRange, float maxRange, bool layout,
+		bool continuous, border_style borderStyle, thumb_style thumbStyle,
+		uint8 resolution)
+	: BBox(borderStyle, NULL)
+	, fMinRange(minRange)
```

Comment:
is this style (comma on the next line) consistent with the rest of the source files?

Reply:
some - it's a copy/pasted bad style.  I fixed a couple of other files with the same issue.